test/index.test.js

```diff
@@ -43,33 +63,6 @@ describe('storage-engine', function () {
       storage.setItem('pew', 'waddup');
     });
 
-    it('stores and fetches no value', async function () {
-      await storage.setItem('noValue');
-      assume(await storage.getItem('noValue')).equals(undefined);
-    });
-
-    it('emits <key> when a no value is get', function (next) {
-      storage.once('noValue', function (method, value) {
-        assume(method).equals('getItem');
-        assume(value).equals(undefined);
-
-        next();
-      });
-
-      storage.getItem('noValue');
```

Comment:
I don't think we have another test for getting a non-existing key, so this test should probably still be kept.

Reply:
yeah, will put back some form of this test.  I'm trying out what happens in actual RN (at least on a simulator...) when calling `getItem` for something that doesn't exist, because the docs are really not clear if it returns null/undefined or a rejected promise :/

Reply:
Seems like it gives a null. for non-existant things. Added back the test to verify that.